src/components/atoms/Navigator/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { memo } from "react";
+
+import { useT } from "@reearth/i18n";
+import { styled } from "@reearth/theme";
+
+import Icon from "../Icon";
+
+import { useNavigator } from "./hooks";
+import { CompassAngle } from "./types";
+
+export type Props = {
+  degree: number;
+  /**
+   * Pass degree of circle as callback arguments.
+   * This event is invoked when outer circle is clicked and mouse move.
+   * If click is dismissed then this event is stopped.
```

Comment:
Sorry what does it mean if "click is dismissed"?

Reply:
Sorry, I fixed this comment...
[chore: fix props description](https://github.com/reearth/reearth-web/pull/308/commits/e547b5673e3bd07cb81c0b2ece3697454c491d1e)